src/install/kubernetes/find-data.mdx

```diff
@@ -0,0 +1,85 @@
+---
+componentType: default
+headingText: Find and use data
+---
+
+
+import kubernetesTestConnection from 'images/kubernetes_screenshot-crop_test-connection.webp'
+
+import kubernetesAddDashboards from 'images/kubernetes_screenshot-crop_add-dashboards.webp'
+
+import kubernetesAddAlerts from 'images/kubernetes_screenshot-crop_add-alerts.webp'
+
+
+Once you've finished all the steps, you can test the connection to make sure everything is working properly and view your data.
+
+<img
+    width="80%;"
+    title="Testing Kubernetes installation"
+    alt="Testing Kubernetes installation"
+    src={kubernetesTestConnection}
+/>
+
+
+<Callout variant="tip">
+    Not seeing data? If you completed the steps and are still not seeing data, check out [this troubleshooting page](/docs/kubernetes-pixie/kubernetes-integration/troubleshooting/kubernetes-integration-troubleshooting-not-seeing-data/).
+</Callout>
+
+After the installation, you should see the following:
```

Comment:
Where will you see this? Where is this information revealed. I think a little bit more info about where they can find this info would be helpful.

Reply:
I've added "in the UI"